pkg/storage/secret/metadata/outbox_store.go

```diff
@@ -28,19 +28,37 @@ func ProvideOutboxQueue(db contracts.Database) contracts.OutboxQueue {
 }
 
 type outboxMessageDB struct {
+	RequestID       string
 	MessageID       string
 	MessageType     contracts.OutboxMessageType
 	Name            string
 	Namespace       string
 	EncryptedSecret sql.NullString
 	KeeperName      sql.NullString
 	ExternalID      sql.NullString
+	ReceiveCount    int
 	Created         int64
 }
 
+type outboxMessageMetadataDB struct {
```

Comment:
Is this being used for anything? I don't see it 👀

Reply:
Nope, forgot to remove it